pkg/handler/processor/dsse/dsse.go

```diff
@@ -17,13 +17,15 @@ package dsse
 
 import (
 	"encoding/base64"
-	"encoding/json"
 	"fmt"
 
 	"github.com/guacsec/guac/pkg/handler/processor"
+	jsoniter "github.com/json-iterator/go"
 	"github.com/secure-systems-lab/go-securesystemslib/dsse"
 )
 
+var json = jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
can you change all iterations of this as needed in the project?

Reply:
there are many packages that use the `"encoding/json"` could you replace them as necessary in this one PR?

Reply:
there are 66 file changes, and making that many changes is hard for me.

Reply:
should be a simple find and replace with the new library. The unit tests are already in place to check it all. Did you have some other concerns?

Reply:
Also, I can help as needed. Let me know :)

Reply:
I understand, I will work on it. Thank you.